hummingbot/client/command/gateway_command.py

```diff
@@ -486,9 +486,18 @@ async def _gateway_connect(self, connector: str = None):
 
     @staticmethod
     async def _fetch_gateway_configs() -> Dict[str, Any]:
-        return await GatewayHttpClient.get_instance().get_configuration(fail_silently=True)
+        try:
+            return await GatewayHttpClient.get_instance().get_configuration(fail_silently=True)
+        except Exception:
+            raise
```

Comment:
I think for this part, we only need to set `fail_silently=False`, without the try-catch blocks